specs/protocol/configurability.md

```diff
@@ -0,0 +1,79 @@
+| [Gas limit](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L298-L300)                             | L2 block gas limit.                                                                                                          | [System Config Owner](#admin-roles)                 |
+| Gas Token                             | Native token used to pay for gas.                                                                                            |                                     |
+| Genesis state                         | Initial state at chain genesis, including code and storage of predeploys (all L2 smart contracts). See [Predeploy](../glossary.md#l2-genesis-block). |             |
+| [L2 block time](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/L2OutputOracle.sol#L105)                         | Frequency with which blocks are produced as a result of derivation.                                                          | [L1 Proxy Admin](#admin-roles)                      |
+| [Resource config](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L338-L340)                       | Config for the EIP-1559 based curve used for the deposit gas market.                                                         | [System Config Owner](#admin-roles)                 |
+| Sequencing window                     | Maximum allowed batch submission gap, after which L1 fallback is triggered in derivation.                                    |                                     |
+| [Start block](https://github.com/ethereum-optimism/optimism/blob/c927ed9e8af501fd330349607a2b09a876a9a1fb/packages/contracts-bedrock/src/L1/SystemConfig.sol#L184)                           | Block at which the system config was initialized the first time.                                                             | [L1 Proxy Admin](#admin-roles)                      |
+| Superchain target                     | Choice of cross-L2 configuration. May be omitted in isolated OP Stack deployments. Includes SuperchainConfig and ProtocolVersions contract addresses. |            |
+
+## Policy Parameters
+
+| Config Property                       | Description                                                                                                                  | Administrator                       |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| Batch submission frequency            | Frequency with which batches are submitted to L1 (see [Batcher Transaction](../glossary.md#batcher-transaction)).            |                                     |
```

Comment:
can we specify that the batcher role is the administrator of this and the compression ratio

Reply:
https://github.com/ethereum-optimism/specs/pull/98/commits/74eb66775dc76013b8288886219dad0185c1f626